builder/tests/builder/src/test/java/io/helidon/builder/test/CustomNamedTest.java

```diff
@@ -32,24 +31,36 @@ class CustomNamedTest {
 
     @Test
     void testIt() throws Exception {
-        DefaultCustomNamed.Builder customNamedBuilder = DefaultCustomNamed.builder()
-                .addStringList("b").addStringList("a").addStringList("b").addStringList("y")
-                .addStringToIntegerMap("b", 1).addStringToIntegerMap("e",2).addStringToIntegerMap("a", 3)
-                .addStringSet("b").addStringSet("a").addStringSet("b").addStringSet("y");
+        CustomNamed.Builder customNamedBuilder = CustomNamed.builder()
+                .addStringList("b")
+                .addStringList("a")
+                .addStringList("b")
+                .addStringList("y")
+                .putStringToIntegerMap("b", 1)
```

Comment:
I know we discuss this to use `put`, but `add` could have been used here, and it makes it better to look at since it would be more consistent.